Dockerfile

```diff
@@ -6,8 +6,8 @@ WORKDIR /app
 
 COPY . .
 
-RUN yarn install
-RUN yarn build
+RUN pnpm
```

Comment:
```suggestion
RUN pnpm install
```

Reply:
also, I think this one is going to fail? 
because default node image does not have `pnpm` by default?